tests/prediction/time_series/data_loader/transfer_learning_data_loader_test.py

```diff
@@ -77,3 +81,42 @@ def test_train_observed_do_not_change_when_called_multiple_times():
     loader.load_data()
     after = loader.train_observed
     assert np.array_equal(before, after)
+
+
+def test_x_test_data_is_exported_when_export_path_is_provided():
+    EXPORT_PATH = "tests/prediction/time_series/test_data/export"
```

Comment:
Why is that in caps lock?

Reply:
Because the export path is a constant, and constants variable should have capital name. See here: https://realpython.com/python-constants/#what-constants-are.

Reply:
Only for module level constants.

Reply:
Changed accordingly. Please look whether this comment is still relevant.